### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by Anan5a** [jellyfin/jellyfin-vue#261] [General]
I am confused, 
How is this different from @jellyfin/jellyfin-web ?

**Comment by camc314** [comment]
See 
https://github.com/jellyfin/jellyfin-web/issues/889
And 
https://github.com/jellyfin/jellyfin-web/issues/1901

**Comment by Extarys** [comment]
@Anan5a Came here after seeing the -web framework migration.

@jellyfin/jellyfin-web > Legacy, current web version
@jellyfin/jellyfin-vue > Being developed, new UI, the future
:rocket: :heart:

**Reply by Bitwolfies** [reply]
Well, the hopeful future, from what I know they make zero promises that this will replace the web client, even agreed they shouldn't call it "jellyfin ui next" or whatever it was gonna be.

**Reply by Extarys** [reply]
Huh, strange... now I'm confused. Maybe it depends how much love the project receive. Thanks for this, I'll control my hype.

**Reply by heyhippari** [reply]
Basically, we're not committing to it because we're never sure how far a project will go. We had a false start with a React Native client at the very start of the project, which made us a bit weary of confirming this kind of things.

I can say, however, that I'm very much committed to this client and there is a lot of hype internally for the direction this is taking, especially with a few other first-time contributors appearing on this project recently and helping me get things done.

**Reply by Extarys** [reply]
Thanks @MrTimscampi . The hype is real here too! I understand your point and I agree. I'm very excited and glad new people are joining - just saw your poll on reddit ;)
